CONTRIBUTING.md

```diff
@@ -92,3 +162,42 @@ We welcome contributions, large or small, including:
 If you would like to contribute new APIs, please check the [CODEOWNERS](https://github.com/iTwin/itwinjs-core/blob/master/.github/CODEOWNERS) file to ensure your contributions align with the package owner's and project goals.
 
 Thank you for taking the time to contribute to open source and making great projects like iTwin.js possible!
+
+## Frequently Asked Questions
+
+### Rush commands take too long, is there a way to speed up my contribution workflow?
+
+If your source code change only impacts the subdirectory you are working on, you can use a package manager to run local commands, without affecting other packages.
+
+Eg. I add a new method within `core/frontend`, also adding a relevant unit test in that folder's `src/test`. I can navigate to the root of that subdirectory, and run `pnpm build`, followed by `pnpm test` or `pnpm cover`.
```

Comment:
Shouldn't the default recommendation be `rushx` instead of `pnpm`?

Reply:
yes, I just fixed it